server/webapp/WEB-INF/rails/app/assets/stylesheets/module.scss

```diff
@@ -2051,7 +2051,7 @@ li.template_form_error {
     width: 100%;
 }
 
-.group_pipelines .move_pipeline span{
+.group_pipelines .for_down_arrow {
```

Comment:
Would rather us not touch the old css, even if it's a "cleanup" job that feels "safe" :)